examples/cra-react15/.storybook/config.js

```diff
@@ -0,0 +1,14 @@
+import { configure, addParameters } from '@storybook/react';
+import { create } from '@storybook/theming';
+
+function loadStories() {
+  require('../src/stories');
+}
+
+addParameters({
+  options: {
+    theme: create({ colorPrimary: 'hotpink', colorSecondary: 'orangered' }),
```

Comment:
Do you think you should comment out the theme to stop this from breaking `next`? Will still make it much easier to test

Reply:
@tmeasday Unfortunately it still breaks, I think due to #6435